docs/common/core/coffee-dto.adoc

```diff
@@ -11,7 +11,8 @@ Több almodulból ál.
 == coffee-dto-base
 Tartalma olyan java osztályok melyek ősként szolgálnak az egész Coffee framework-ben.
 
-Itt táláható a java.time (java 8+) XSD univerzális adapterek is.
+Itt táláható a java.time (java 8+) XSD univerzális adapterek és rest végpontokhoz tartozó alap path-okat kezelő osztály.
```

Comment:
Lehet azt is meg kéne említeni hogy erre a verzióra váltásnál meg kell lépni azt is hogy az importoknál módosítani kell a BaseServicePath-ot.

Reply:
Igen jogos, migracios doksi lemaradt